src/models/serverRoomIDModels.ts

```diff
@@ -62,6 +62,10 @@ const roomIdSchema: Schema = new Schema(
       type: String,
       required: true,
     },
+    updatedBy: {
+      type: String,
```

Comment:
The string should be in lowercase.

Reply:
No , it's should be the same in type script in upperCase